lib/index.js

```diff
@@ -11,18 +11,18 @@ const logger = pino({
   }
 });
 
-const ns = createNamespace(NAMESPACE);
+const namespace = createNamespace(NAMESPACE);
 
 const getRequestId = () => {
-  return ns ? ns.get('requestId') : null;
+  return namespace ? namespace.get('requestId') : null;
 };
 
 const setRequestId = (id = uuid()) => {
-  ns.set('requestId', id);
+  namespace.set('requestId', id);
   return id;
 }
 
-const prependRequestId = msg => {
+const prependRequestId = (msg) => {
```

Comment:
This change is required for eslint ?

Reply:
I changed it when I was trying out solutions and forgot to change it back, nice catch.